modules/news/news.service.js

```diff
@@ -36,7 +41,7 @@ class NewsService {
         },
       },
     });
-    return newsCount > 0;
+    return newsCount;
```

Comment:
Revert this change, because this method performs check and returns a boolean value but not the count

Reply:
Changed!